src/App.tsx

```
@@ -1,10 +1,106 @@
+// ❌ Function with boolean parameter
+function process(data: any, shouldValidate: boolean) {
+  // ❌ Generic function name "process"
+  // ❌ Boolean parameter
+  if (shouldValidate) {
+    return validateData(data);
+  }
+  return data;
+}
+
+// ❌ Function not following naming conventions
+function validateData(data: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)